wd_helper.au3

```diff
@@ -475,7 +475,7 @@ Func _WD_WaitScript($sSession, $sJavaScript, $iDelay = Default, $iTimeout = Defa
 	Local Const $sFuncName = "_WD_WaitScript"
 	Local Const $sParameters = 'Parameters:   JavaScript=' & $sJavaScript & '   Delay=' & $iDelay & '   Timeout=' & $iTimeout
 	Local $iErr
-	Local $sValue = False
+	Local $bValue = False
```

Comment:
it is too common name for variable especially in the case when you are in fact not trying to get value but to test

please rename it to `$bResult` or `$bTest`

Reply:
@mlipok I think you are being too picky here. The variable is declared locally, so I don't see how giving it a common name could cause any issues. Technically, `true` and `false` *are* values. 😉 

Please elaborate if you disagree.